ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -6,6 +6,22 @@
 
 {% block content %}
 
+<style>
+  @keyframes blink {
+    0% { opacity: 0; }
```

Comment:
Question: Why change the opacity? Can't you use visibility/visible?

Reply:
it didn't seem to work when I tried that haha

Reply:
I could look more into it but this worked and it wasn't anything major so I went with it